include/openPMD/binding/python/Mpi.hpp

```diff
@@ -0,0 +1,100 @@
+/* Copyright 2021 Axel Huebl and Franz Poeschel
+ *
+ * This file is part of openPMD-api.
+ *
+ * openPMD-api is free software: you can redistribute it and/or modify
+ * it under the terms of of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * openPMD-api is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with openPMD-api.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "openPMD/config.hpp"
+
+#if openPMD_HAVE_MPI
+
+#include <mpi.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
```

Comment:
Please use `#include "openPMD/binding/python/Common.hpp"` instead to avoid one-definition-rule violations in pybind11.